ui/src/css/variables.sass

```diff
@@ -584,8 +584,8 @@ $toolbar-title-letter-spacing   : .01em !default
 $toolbar-title-padding          : 0 12px !default
 
 $layout-border                  : 1px solid $separator-color !default
-$layout-shadow                  : 0 0 10px 2px rgba(0,0,0,0.2), 0 0px 10px rgba(0,0,0,0.24) !default
-$layout-shadow-dark             : 0 0 10px 2px rgba(255,255,255,.2), 0 0px 10px rgba(255,255,255,.24) !default
+$layout-shadow                  : 0 0 10px 2px rgba($shadow-color,0.2), 0 0px 10px rgba($shadow-color,0.24) !default
```

Comment:
I don't think that you can use `hex` values (`$shadow-color: #000`, `$dark-shadow-color: #FFF`) inside a `rgba` color definition.

Reply:
On the contrary, you can use hex values in variables this way. Here is working example in [codepen](https://codepen.io/popadicbranislav/pen/XWBwMxW)

Reply:
You are correct, I apologize for the mistake.